jib-gradle-plugin/CHANGELOG.md

```diff
@@ -23,30 +23,30 @@ All notable changes to this project will be documented in this file.
 
 ### Changed
 
-- To disable parallel execution, the property `jib.serialize` should be used instead of `jibSerialize`. ([#1968](https://github.com/GoogleContainerTools/jib/issues/1968))
-- For retrieving credentials from Docker config (`~/.docker/config.json`), `credHelpers` now takes precedence over `credsStore`, followed by `auths`. ([#1958](https://github.com/GoogleContainerTools/jib/pull/1958))
-- The legacy `credsStore` no longer requires defining empty registry entries in `auths` to be used. This now means that if `credsStore` is defined, `auths` will be completely ignored. ([#1958](https://github.com/GoogleContainerTools/jib/pull/1958))
-- `jib.dockerClient` is now configurable on all tasks, not just `jibDockerBuild`. ([#1932](https://github.com/GoogleContainerTools/jib/issues/1932))
+- To disable parallel execution, the property `jib.serialize` should be used instead of `jibSerialize` ([#1968](https://github.com/GoogleContainerTools/jib/issues/1968))
+- For retrieving credentials from Docker config (`~/.docker/config.json`), `credHelpers` now takes precedence over `credsStore`, followed by `auths` ([#1958](https://github.com/GoogleContainerTools/jib/pull/1958))
+- The legacy `credsStore` no longer requires defining empty registry entries in `auths` to be used. This now means that if `credsStore` is defined, `auths` will be completely ignored ([#1958](https://github.com/GoogleContainerTools/jib/pull/1958))
```

Comment:
Not that I care much either, but I wonder if we should add dots to all of them instead of removing, since now the multi-sentence lines look weird (all sentences end with a period except the last one).

Reply:
Yeah, I like it. I keep forgetting removing dots.